tests/calculator.py

```diff
@@ -0,0 +1,7 @@
+x = input("Enter the first number: ")
+y = input("Enter the second number: ")
+
+print("\nThe sum is:",int(x) + int(y))
+print("The difference is:", int(x)- int(y))
+print("The multiplication is: ", int(x) * int(y))
+print("The division is: ", float(x)/float(y))
```

Comment:
Stai eseguendo il casting a int/float molteplici volte, fallo una volta sola in fase di definizione.